src/SettingsView/EventQueryListItem.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import IconButton from 'material-ui/IconButton';
+import IconEdit from 'material-ui/svg-icons/editor/mode-edit';
+import IconDelete from 'material-ui/svg-icons/action/delete';
+import ConfirmationMessage from './../Utils/ConfirmationMessage.js'
+import {ListItem} from 'material-ui/List';
+import config from './../config/config.js';
+
+
+class EventQueryListItem extends Component {
+	
+	constructor(props) {
+		super(props);
+		this.deleteEventQuery = this.deleteEventQuery.bind(this);
+		this.deleteEventQueryButton = this.deleteEventQueryButton.bind(this);
+	}
+	
+	deleteEventQuery() {
+		this.props.deleteQuery(this.props.query);
+	}
+
+	editEventQuery() {
+		// TODO: to be filled
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1135)